Packs/JSONSampleIncidentGenerator/Integrations/JSONSampleIncidentGenerator/JSONSampleIncidentGenerator.py

```diff
@@ -59,3 +72,5 @@ def main() -> None:
 ''' ENTRY POINT '''
 if __name__ in ('__main__', '__builtin__', 'builtins'):
     main()
+
+
```

Comment:
`JSONSampleIncidentGenerator.py:76:1: W391 blank line at end of file
`
please delete line 76